tests/unittests/tests-hashes/tests-hashes-sha256.c

```diff
@@ -1,12 +1,27 @@
 /*
- * Copyright (C) 2014 Philipp Rosenkranz <philipp.rosenkranz@fu-berlin.de>
  * Copyright (C) 2013 Christian Mehlis <mehlis@inf.fu-berlin.de>
+ * Copyright (C) 2014 Philipp Rosenkranz <philipp.rosenkranz@fu-berlin.de>
```

Comment:
I hope its ok that I reordered the copyrights by date ascending with each new entry